src/app/editor.js

```diff
@@ -2,10 +2,32 @@
 
 var EventManager = require('../lib/eventManager')
 
+var csjs = require('csjs-inject')
 var ace = require('brace')
 var Range = ace.acequire('ace/range').Range
 require('../mode-solidity.js')
 
+var css = csjs`
```

Comment:
sure - but honestly - is this a good idea?
I feel, since I started to put the css related to a certain component into the place of that component, that things became a lot easier.

Having tons of global variables is a bad practice and that is what a css with tons of global selectors is. The global stylesheet will grow longer and longer and people will start adding hacks to it that interfere with each other until it becomes really really hard to change styling, especially layout related stuff.

That's my experience.